### PR TITLE
OSDOCS-3247: Updated cluster-wide proxy prereqs

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR adds the full set of AWS prerequisites to the cluster-wide proxy documentation.

JIRA: https://issues.redhat.com/browse/OSDOCS-3247

PREVIEW: https://deploy-preview-41726--osdocs.netlify.app/openshift-rosa/latest/networking/configuring-cluster-wide-proxy.html#prerequisites

### CODE REVIEW

File: networking/configuring-cluster-wide-proxy.adoc [hunk 1]
```diff
@@ -15,6 +15,22 @@ You can configure a cluster-wide proxy during cluster installation.
 
 * You are the cluster owner.
 * Your account has sufficient privileges.
+* You have an existing `install-config.yaml` file.
+* You reviewed the sites that your cluster requires access to and determined whether any of them need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to hosting cloud provider APIs. You added sites to the `Proxy` object's `spec.noProxy` field to bypass the proxy if necessary.
++
```
Comment (mrbarge) [inline networking/configuring-cluster-wide-proxy.adoc:20]: These lines can all be removed - they're not relevant for the managed service offering.

Comment (EricPonvelle) [reply networking/configuring-cluster-wide-proxy.adoc]: @mrbarge I removed lines 18 and 19. Are we good to merge this?

Comment (mrbarge) [reply networking/configuring-cluster-wide-proxy.adoc]: Thank you! This looks fine to me.

**Review by adellape — COMMENTED** [review]
Looks good tho I couldn't confirm the appearance of the `aws` and `aws-china` ifevals.

**Pull request merged**